src/app/components/shopping-cart/product-list/product-list.component.html

```diff
@@ -0,0 +1,63 @@
+<marquee behavior="scroll" direction="left">
+    <img src="../../../../assets/offer1.jpg" alt="Offer1" class="banner">
+    <img src="../../../../assets/offer2.jpg" alt="Offer1" class="banner">
+    <img src="../../../../assets/offer3.jpg" alt="Offer1" class="banner">
+    <img src="../../../../assets/offer4.jpg" alt="Offer1" class="banner">
+    <img src="../../../../assets/offer5.jpg" alt="Offer1" class="banner">
+</marquee>
+<div class="border"></div>
+<div class="row">
+    <div class="col">
+        <img src="../../../../assets/baby.png" alt="Baby Items" class="baby">
+        <div class="baby-info">
+            <h4>Baby Care</h4>
+            <small>Shop online for Baby Products, Diapers,Skin Care Products, etc</small><br>
+            <button class="baby-btn">Explore Baby</button>
+        </div>
+    </div>
+</div>
+<hr>
+<div class="row">
+    <div class="col">
+        <img src="../../../../assets/bakery.png" alt="Bakery Items" class="bakery">
+        <div class="bakery-info">
+            <h4>Bakery cakes and dairy</h4>
+            <small>The best cup cake, cookies, muffins</small><br>
+            <button class="bakery-btn">Explore Bakery-cakes-dairy</button>
+        </div>
+    </div>
+</div>
+<hr>
+<div class="row">
+    <div class="col">
+        <img src="../../../../assets/beverages.png" alt="Beverages Items" class="beverages">
+        <div class="beverages-info">
+            <h4>Beverages</h4>
+            <small>Our beverage department will ensure your fridge is always fully stocked. Shop for Soda, juice, beer and more</small><br>
+            <button class="beverages-btn">Explore Beverages</button>
+        </div>
+    </div>
+</div>
+<hr>
+<div class="row">
+    <div class="col">
+        <img src="../../../../assets/fruits.png" alt="Beverages Items" class="fruits">
+        <div class="fruits-info">
+            <h4>Fruits and Vegetables</h4>
+            <small>Explore variety of fruits and vegetables</small><br>
+            <button class="fruits-btn">Explore fruits and veg</button>
+        </div>
+    </div>
+</div>
+<hr>
+<div class="row">
+    <div class="col">
+        <img src="../../../../assets/beauty.png" alt="Beauty Items" class="beauty">
+        <div class="beauty-info">
+            <h4>Beauty and Hygeine</h4>
+            <small>Buy beauty and personal care products online in India at best price</small><br>
+            <button class="beauty-btn">Explore Beauty-Hygeiene</button>
+        </div>
+    </div>
+</div>
+<br>
```

Comment:
should be dynamic as coming from the api response

Reply:
I have worked on this and will commit the code shortly.